### PR TITLE
doc: faq gpu compatibility

### PR DESCRIPTION
**Pull request by BruceMacD**
Add some information about GPU compatibility to the FAQs.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ curl -fsSL https://ollama.com/install.sh | sh
 
 Review the [Troubleshooting](./troubleshooting.md) docs for more about using logs.
 
+## Is my GPU compatible with Ollama?
+
+### Nvidia
+Ollama supports Nvidia GPUs with compute capability 3.5 to 8.6.
```
Comment (dhiltgen) [inline docs/faq.md:20]: 5.0 is our current minimum.

I have a PR open that would add 3.5, but I don't intend to merge it as is, and at some point I was planning to refactor it to allow building from source to get 3.5+ support but not ship it officially.  In testing, what I see is the 3.5 GPUs are slower than modern CPUs, and to pick them up would either require another variant, or changing compile flags to llama.cpp that may have a negative impact on modern GPUs.

File: docs/faq.md [hunk 2]
```diff
@@ -14,6 +14,20 @@ curl -fsSL https://ollama.com/install.sh | sh
...
+
+Check this table to see if your card is supported:
+[https://en.wikipedia.org/wiki/CUDA#GPUs_supported](https://en.wikipedia.org/wiki/CUDA#GPUs_supported)
```
Comment (dhiltgen) [inline docs/faq.md:23]: What do you think about including a link to this page which I've found super helpful as a decoder - https://developer.nvidia.com/cuda-gpus

**Comment by mchiang0610** [comment]
Thank you for creating this! So awesome. Possible to list the graphics cards by name instead of having people search on their own. Here is what I posted for the NVIDIA release of what's supported: 

GeForce GTX TITAN X 980Ti 980 970 960 950 750Ti 750

GeForce GTX 980M 970M 965M 960M 950M 860M 850M

GeForce 940M 930M 910M 840M 830M

Quadro M5500M M5000 M2200 M1200 M620 M520

Tesla M60 M40

NVS 810

**Comment by BruceMacD** [comment]
> Thank you for creating this! So awesome. Possible to list the graphics cards by name instead of having people search on their own.

I would rather list them too, but there are some cards like the nvidia tesla T40 that have had issues supported, which their compute compatibility makes it look like they should work. Might be a system specific driver issue though.

**Comment by BruceMacD** [comment]
I took a shot at creating a table of supported nvidia gpus

File: docs/gpu.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+| gfx906 | Radeon Instinct MI50 |
+| gfx908 | Radeon Instinct MI100 |
+| gfx90a | Radeon Instinct MI210 |
+| gfx940 | Radeon Instinct MI300 |
+| gfx941 | |
+| gfx942 | |
+| gfx1030 | Radeon PRO V620 |
+| gfx1100 | Radeon PRO W7900 |
+| gfx1101 | Radeon PRO W7700 |
+| gfx1102 | Radeon RX 7600 |
```
Comment (dhiltgen) [inline docs/gpu.md:144]: For 0.1.29, Jeff and Michael built out a more exhaustive list of Radeon cards that should be supported.  We might want to inject that here.

**Pull request merged**